src/app/my-account/my-account/my-account.component.ts

```diff
@@ -44,7 +53,34 @@ export class MyAccountComponent implements OnInit {
     dialogRef.componentInstance.onClosed.pipe(take(1)).subscribe((confirmed: boolean) => {
       dialogRef.close();
       if (confirmed) {
-        // TODO: call to api
+        if (announcement.type === TransportOffer.TypeEnum.Transport) {
+          this.transportResourceService
+            .deleteTransport(announcement.id!)
+            .pipe(
+              switchMap((_) => {
+                return this.myOffersResource.listMyOffers(this.pageRequest);
+              })
+            )
+            .subscribe((data) => (this.myAnnouncements = data));
+        } else if (announcement.type === AccommodationOffer.TypeEnum.Accommodation) {
+          this.accommodationsResourceService
+            .deleteAccommodations(announcement.id!)
+            .pipe(
+              switchMap((_) => {
+                return this.myOffersResource.listMyOffers(this.pageRequest);
+              })
+            )
+            .subscribe((data) => (this.myAnnouncements = data));
+        } else {
+          this.materialAidResourceService
```

Comment:
I would reccomand another if else, to limit strictly matierialAid, because it should not be open for any other offerTypes